packages/terra-form/src/components/ChoiceField.jsx

```diff
@@ -0,0 +1,103 @@
+/* eslint-disable jsx-a11y/label-has-for */
+
+import React, { PropTypes } from 'react';
+
+import Control from './Control';
+import Field from './Field';
+
+const propTypes = {
+  /**
+   * Choices to populate radio buttosn for
+   */
+  choices: PropTypes.array,
```

Comment:
I believe Brett called out on the original Tech Design that this prop might make more sense being called `options`.

I personally don't think we should use a prop API that consists of an Array of hashes with key value pairs. The React form libraries I've worked with use children and I think it makes it a lot easier for the consumer. 

```
      <RadioGroup onChange={this.onChange} value={this.state.value}>
        <Radio value={1}>A</Radio>
        <Radio value={2}>B</Radio>
        <Radio value={3}>C</Radio>
        <Radio value={4}>D</Radio>
      </RadioGroup>
```
https://ant.design/components/radio/